examples/plot_base_plottester-2.py

```diff
@@ -0,0 +1,187 @@
+"""
+The Basics of Testing Plots
+===========================
+
+These are some examples of using the basic functionality of MatPlotCheck.
+
+"""
+
+###############################################################################
+# Setup
+# -----
+# We will start by importing the required packages. We will be using
+# ``matplotlib.pyplot`` to create our plots, but any ``matplotlib`` based
+# plotter (such as ``pandas.DataFrame.plot``) can be used.
+
+import matplotlib.pyplot as plt
+import matplotcheck.base as mpc
+import matplotcheck.notebook as nb
+import pandas as pd
+
+###############################################################################
+# Plot
+# ----
+# Now we'll create some data and plot it.
+
+months = [
+    "Jan",
+    "Feb",
+    "Mar",
+    "Apr",
+    "May",
+    "Jun",
+    "Jul",
+    "Aug",
+    "Sep",
+    "Oct",
+    "Nov",
+    "Dec",
+]
+percip = [
+    0.75,
+    0.83,
+    2.2,
+    2.87,
+    2.80,
+    2.20,
+    1.77,
+    1.85,
+    1.69,
+    1.54,
+    1.22,
+    0.94,
+]
+
+fig, ax = plt.subplots()
+ax.bar(months, percip, color="blue")
+ax.set(
+    title="Average Monthly Percipitation in Boulder, CO",
+    xlabel="Month",
+    ylabel="Percipitation (in)",
+)
+
+plot_1_hold = nb.convert_axes(plt, which_axes="current")
+###############################################################################
+#
+# .. note::
+#    To use matplotcheck to test a plot, we need a copy of the
+#    ``Matplotlib.axis.Axis`` object that the plot is stored in. This can be
+#    prove difficult when testing plots in a Jupyter Notebook, where a
+#    ``Matplotlib.axis.Axis`` object will not persisit beyond the cell it was
+#    created in. We'll use ``nb.convert_axes()`` to hold the most recently
+#    created plot in the variable ``plot_1_hold``. There are other ways to do
+#    this, but in a Jupyter Notebook you *must* use ``nb.convert_axes()`` to
+#    save the ``Axis`` object.
+#
+# Testing the plot
+# ----------------
+# Now we'll use matplotcheck to check that the plot has certain attributes that
```

Comment:
@ryla5068 when we write vignettes and lessons we use "you" instead of "we" . it's just an earth lab style thing. will you please update that throughout?